pkg/engine/apicall/apiCall.go

```diff
@@ -82,17 +87,15 @@ func (a *apiCall) Execute(ctx context.Context, call *kyvernov1.APICall) ([]byte,
 }
 
 func (a *apiCall) transformAndStore(jsonData []byte) ([]byte, error) {
```

Comment:
How do we handle `jsonData==nil` case?

Reply:
From my point of view, this function will not be called if jsonData == nil as we handle this error in the fetch function and return an error there if no default value is defined. 

But if you recommend double checking within this function again for jsonData == nil, what should be the return values? nil, nil will should let the next function fail, no?